.github/workflows/pr-realm-js.yml

```diff
@@ -0,0 +1,391 @@
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Cache prebuild artifacts
+        id: cache-prebuild
+        uses: actions/cache@v2
+        env:
+          cache-name: prebuild
+        with:
+          path: ${{ matrix.variant.artifact-path }}
+          key: ${{ runner.os }}-${{ matrix.variant.os }}-${{ matrix.variant.arch }}-${{ env.cache-name }}-${{hashFiles('./src/**', './vendor/**')}}
+
+      - name: MSVC Setup
+        if: ${{ runner.os == 'Windows' }}
+        uses: ilammy/msvc-dev-cmd@v1
+
+      - name: Setup Vcpkg
+        if: ${{ runner.os == 'Windows' }}
+        uses: friendlyanon/setup-vcpkg@v1
+        with:
+          path: vendor/realm-core/tools/vcpkg/ports
+          cache-key: vcpkg-windows-${{ matrix.variant.arch }}
+          cache-restore-keys: vcpkg-windows-${{ matrix.variant.arch }}
+
+      # ninja-build is used by default if available and results in faster build times
+      # On linux, electron requires a connected display.  We fake this by giving it a headless environment using xvfb
+      # Relevant issue: https://github.com/juliangruber/browser-run/issues/147
+      - name: Linux Environment setup
+        if: ${{ (matrix.variant.runner == 'ubuntu-20.04') }}
+        run: sudo apt-get install ccache ninja-build xvfb
+
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
+        #TODO: figure out using ccache or equivalent on windows
+        if: ${{ runner.os != 'Windows' }}
+        with:
+          key: ${{ runner.os }}-${{ matrix.variant.os }}-${{ matrix.variant.arch }}
+          max-size: '2.0G'
+
+      - name: Prepend ccache executables to the PATH
+        if: ${{ runner.os != 'Windows' }}
+        run: |
+          echo "/usr/lib/ccache:/usr/local/opt/ccache/libexec" >> $GITHUB_PATH
+
+      - name: Configure ccache
+        if: ${{ runner.os != 'Windows' }}
+        run: |
+          ccache --set-config="compiler_check=content"
+          ccache --show-config
+
+      - name: Install deps
+        run: npm ci --ignore-scripts
+
+      # Install node dependencies
+      - name: Install prebuilds
+        if: steps.cache-prebuild.outputs.cache-hit == 'true'
+        run: npm ci
+
+      # build the c++ library for standard targets
+      - name: Build node
+        if: ${{ (matrix.variant.os != 'ios') && (matrix.variant.os != 'android') }}
+        run: npm run prebuild:node -- --arch=${{matrix.variant.arch}}
+
+      # build the c++ library for IOS
+      - name: Build IOS
+        if: ${{ (matrix.variant.os == 'ios') }}
+        run: npm run prebuild:ios
+
+      # build the c++ library for Android
+      - name: Build Android
+        if: ${{ (matrix.variant.os == 'android') }}
+        run: npm run prebuild:android -- --arch=${{matrix.variant.arch}}
+
+      - name: Install the node dependencies in the test folder
+        working-directory: ./tests
+        run: npm ci --ignore-scripts
+
+      # all tests passed, upload artifacts
```

Comment:
This says all tests passed but it's not obvious to me that there are tests running as part of the build phase.

Reply:
This can be removed.  There were tests here in the past